server/src/routes/weburl.ts

```diff
@@ -29,13 +29,24 @@ router.post('/create', async (req, res) => {
     }
 })
 
+router.delete('/delete', async (req, res) => {
```

Comment:
I was thinking of having URL in the format `/delete/id`. So basically client will pass the id in the URL params and on the server side we need to delete the shorty on the basis of passed `id`. Passing `body` doesn't seem right to me.

Reply:
@sprakash57 I have updated and pushed changes. Let me know if these updates work fine 👍